gcloud_requests/proxy.py

```diff
@@ -76,31 +64,30 @@ def __init__(self, credentials=None, logger=None):
     def __del__(self):
         _credentials_watcher.unwatch(self.credentials)
 
-    def request(self, uri, method="GET", body=None, headers=None, redirections=5, connection_type=None, retries=0, refresh_attempts=0):   # noqa
+    def request(self, method, url, data=None, headers=None, retries=0, refresh_attempts=0, **kwargs):
```

Comment:
Why drop the default arg here? Personally, I agree with the change, but I worry about potentially breaking other things in our stuff.

Reply:
The order of arguments changed with the new function signature, and `method` has no default.  I think it's best if we stick to what the new API needs and fix anything that was using this directly.

For most use cases we are just overriding the `Client` object's `_http` param: https://googleapis.github.io/google-cloud-python/latest/datastore/client.html#google.cloud.datastore.client.Client